cogs/commands/onmessage.py

```diff
@@ -56,6 +60,22 @@ async def thanks(self, message: Message):
             search(r"\b" + thank + r"\b", message.content.lower()) for thank in thanks
         ):
             return
+
+        # get all command prefixes
+        command_prefixes = self.bot.command_prefix(self.bot, message)
+        # if message is not command, check for karma
+        if not any(message.content.startswith(prefix) for prefix in command_prefixes):
+            # process karma if apropriate
+            if search(r"\+\+|--|\+\-", message.content):
+                reply = process_karma(
```

Comment:
karma processing is done in `../database.py` so this will cause all karma to be double counted